subgen.py

```diff
@@ -504,13 +510,25 @@ async def asr(
         delete_model()
     
     if result:
-        return StreamingResponse(
-            iter(result.to_srt_vtt(filepath=None, word_level=word_level_highlight)),
-            media_type="text/plain",
-            headers={
-                'Source': 'Transcribed using stable-ts from Subgen!',
-            }
-        )
+        if output == 'srt':
+          return Response(
```

Comment:
Reluctant to implement response instead of StreamingResponse.  I see no downside to StreamingResponse whereas Response will start to have performance issues on file exceeding 10-50mb.

Reply:
StreamingResponse was really slow delivering the response, changing to Response made it much more performant. It was just a quick solution for me, don't read much into it.